spec/src/main/asciidoc/retry.asciidoc

```diff
@@ -39,6 +39,29 @@ If applied to a class, it means the all methods in the class will have the `@Ret
 If applied to a method, it means  that method will have `@Retry` policy applied.
 If the `@Retry` policy applied on a class level and on a method level within that class, the method level `@Retry` will override the class-level `@Retry` policy for that particular method.
 
+When a method returns and the retry policy is present, the following rules are applied:
+
+* If the method returns normally (doesn't throw), the result is simply returned.
+* Otherwise, if the thrown object is assignable to any value in the `abortOn` parameter, the thrown object is rethrown.
+* Otherwise, if the thrown object is assignable to any value in the `retryOn` parameter, the method call is retried.
+* Otherwise the thrown object is rethrown.
+
+For example, to retry on all exceptions except of IO exceptions, one would write:
```

Comment:
Grammar: replace "except of" with "except for"

Reply:
This sounds weird to me, but I'll trust your English more than mine :-)

Reply:
Heh, that's fair enough. For what it's worth, I've never heard "except of" used, but any of the following would be fine:
* all exceptions *except* IO exceptions
* all exceptions *except for* IO exceptions
* all exceptions *other than* IO exceptions
* all exceptions *apart from* IO exceptions
* all exceptions *with the exception of* IO exceptions